Update Readme for Keyrings (#156)
aws/aws-encryption-sdk-java: README.md

The following code sample demonstrates how to get started:

1. Instantiate the SDK.
2. Setup an AWS KMS Keyring.
3. Encrypt and decrypt data.

```java
// This sample code encrypts and then decrypts data using an AWS KMS customer master key (CMK).
package com.amazonaws.crypto.examples;

import java.nio.charset.StandardCharsets;
import java.util.Arrays;
import java.util.Collections;
import java.util.Map;

import com.amazonaws.encryptionsdk.AwsCrypto;
import com.amazonaws.encryptionsdk.AwsCryptoResult;
import com.amazonaws.encryptionsdk.DecryptRequest;
import com.amazonaws.encryptionsdk.EncryptRequest;
import com.amazonaws.encryptionsdk.keyrings.Keyring;
import com.amazonaws.encryptionsdk.keyrings.StandardKeyrings;
import com.amazonaws.encryptionsdk.kms.AwsKmsCmkId;

public class BasicEncryptionExample {

    private static final byte[] EXAMPLE_DATA = "Hello World".getBytes(StandardCharsets.UTF_8);

    public static void main(final String[] args) {
        encryptAndDecrypt(AwsKmsCmkId.fromString(args[0]));
    }

    static void encryptAndDecrypt(final AwsKmsCmkId keyArn) {
        // 1. Instantiate the SDK
        final AwsCrypto crypto = new AwsCrypto();

>         // 2. Instantiate an AWS KMS Keyring, supplying the key ARN as the generator for generating a 

Reviewer: ```suggestion
        // 2. Instantiate a KMS keyring. Supply the key ARN for the generator key that generates a 
```

        //    data key. While using a key ARN is a best practice, for encryption operations it is also

>         //    acceptable to use a CMK alias or an alias ARN.

Reviewer: While using a key ARN is a best practice, for encryption operations you can also
        //    use an alias name or alias ARN.

        final Keyring keyring = StandardKeyrings.awsKms(keyArn);

        // 3. Create an encryption context
        //
        //    Most encrypted data should have an associated encryption context
        //    to protect integrity. This sample uses placeholder values.
        //
        //    For more information see: https://amzn.to/1nSbe9X (blogs.aws.amazon.com)
        final Map<String, String> encryptionContext = Collections.singletonMap("Example", "String");

        // 4. Encrypt the data with the keyring and encryption context
        final AwsCryptoResult<byte[]> encryptResult = crypto.encrypt(
                EncryptRequest.builder()
                    .keyring(keyring)
                    .encryptionContext(encryptionContext)
                    .plaintext(EXAMPLE_DATA).build());
        final byte[] ciphertext = encryptResult.getResult();

>         // 5. Decrypt the data. The same keyring may be used to encrypt and decrypt, but for decryption

Reviewer: ```suggestion
        // 5. Decrypt the data. You can use the same keyring to encrypt and decrypt, but for decryption
```
